acir/src/circuit/opcodes.rs

```diff
@@ -7,21 +7,13 @@ use crate::serialization::{
 };
 use crate::BlackBoxFunc;
 use acir_field::FieldElement;
+use brillig_bytecode::ForeignCallResult;
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize, Debug)]
 pub enum BrilligInputs {
     Simple(Expression),
-    Array(u32, Vec<Expression>),
-}
-
-impl BrilligInputs {
-    fn to_u16(&self) -> u16 {
-        match self {
-            BrilligInputs::Simple(_) => 0,
-            BrilligInputs::Array { .. } => 1,
-        }
-    }
+    Array(Vec<Expression>),
```

Comment:
I removed the ID because I didn't really see where to use it, does this cause issues for SSA?